### PR TITLE
Removed Spark2.2 app and updated MapR Document

### PR DESCRIPTION
**Pull request by abhinandantn**
- Removed Spark 2.2 app since Spark 2.4.5 is already available
- Updated MapR document with location of the docker images.

### CODE REVIEW

**Review by riteshja — APPROVED** [review]

File: deploy/example_catalog/docs/mapr610-readme.md [hunk 1]
```diff
@@ -117,4 +117,11 @@ HttpFS is used to list the content of maprfs. It can be used 2 ways, through bro
...
-        curl -u mapr -i "http(s)://<link given in the service endpoints:port as per service endpoints>/webhdfs/v1/user?op=LISTSTATUS&user=mapr"
\ No newline at end of file
+        curl -u mapr -i "http(s)://<link given in the service endpoints:port as per service endpoints>/webhdfs/v1/user?op=LISTSTATUS&user=mapr"
+        
+#### Docker image location
+
+There are 2 images used in the app and the following are the location of images.
+
+docker.io/bluedata/mapr610:1.3
+docker.io/bluedata/mapr610edge:1.3
```
Comment (kmathur2) [inline deploy/example_catalog/docs/mapr610-readme.md:127]: what is the edge image for? Do we have a edge role in mapr app ?

File: deploy/example_clusters/cr-cluster-spark221e2-stor.yaml [hunk 2]
```diff
@@ -1,37 +0,0 @@
-apiVersion: "kubedirector.hpe.com/v1beta1"
-kind: "KubeDirectorCluster"
```
Comment (kmathur2) [inline deploy/example_clusters/cr-cluster-spark221e2-stor.yaml:2]: why are we removing this? We need to remove this from Everest also

File: deploy/example_catalog/docs/mapr610-readme.md (hunk 1 above)
Comment (riteshja) [reply deploy/example_catalog/docs/mapr610-readme.md:127]: yes there is an edge role

File: deploy/example_clusters/cr-cluster-spark221e2-stor.yaml (hunk 2 above)
Comment (riteshja) [reply deploy/example_clusters/cr-cluster-spark221e2-stor.yaml:2]: yes. i think there is a PR in everest for removing this. This image is buggy and we have newer spark 2.4.5 available in KD.

**Review by kmathur2 — APPROVED** [review]

**Pull request merged**